idmega2000/visitrail
# #166804623 user authentication system

**What does this PR do?**
It implements the user authentication system of the application

**Description of Task to be completed?**
- create user login endpoint
- create user signup endpoint
- creat get and update user endpoint

**How should this be manually tested?**
- Fetch the branch `git fetch ft-user-authentication-166804623`
- Check out into the branch `ft-user-authentication-166804623`
- run python manage.py migrate
- run python manage.py runserver
- signup hitting the /signup endpoint
- you should be able to login with the signup details
Any pertinent information that should be considered

**What are the relevant pivotal tracker stories?**
[166804623](https://www.pivotaltracker.com/story/show/166804623)

**Screenshots (if appropriate)**
None
